### DISCUSSION TITLE
ArrayTable 和 ahooks 中的 useRequest 共用，导致 form.setInitialValues 失效

### DISCUSSION THREAD

**Discussion by kongjunAI** [alibaba/formily#3710] [Q&A 2.x]
ArrayTable 和 ahooks 中的 useRequest 共用，导致 form.setInitialValues 失效

![image](https://user-images.githubusercontent.com/76086371/218100170-0ff111a2-a836-436e-a59c-425dea33ec4b.png)

**Comment by xiaosi-ss** [comment]
observer 包一下就可以吧
